composer.json

```diff
@@ -269,6 +269,8 @@
         ],
         "integration-tests": [
             "@composer create-test-db",
+            "@composer install --working-dir=modules/ps_apiresources",
+            "@php -d date.timezone=UTC ./vendor/phpunit/phpunit/phpunit -c modules/ps_apiresources/tests/Integration/phpunit.xml",
```

Comment:
Can you run this part after the core ones please? We probably want feedback from the core tests first in the CI (especially if we need to run the composer install in the module, this could hide unexpected bugs in the core for weird reasons)